gateleen-security/src/main/java/org/swisspush/gateleen/security/authorization/Authorizer.java

```diff
@@ -67,47 +63,42 @@ public Authorizer(Vertx vertx, final ResourceStorage storage, String securityRoo
         this.aclRoot = securityRoot + aclKey + "/";
         this.aclUriPattern = new PatternHolder(Pattern.compile("^" + aclRoot + "(?<role>.+)$"));
         this.userUriPattern = Pattern.compile(securityRoot + "user(\\?.*)?");
+        this.roleMapperUriPattern = new PatternHolder(Pattern.compile("^" + securityRoot + "/rolemapper"));
         this.roleExtractor = new RoleExtractor(rolePattern);
-
-        this.aclSchema = ResourcesUtils.loadResource("gateleen_security_schema_acl", true);
-        this.aclFactory = new AclFactory(aclSchema);
+        this.roleMapper = new RoleMapper(storage,securityRoot);
```

Comment:
code format - need space after comma.
This is not a blocker for me - still, if you touch the code again, please format to avoid future diffs